components/p3a/BUILD.gn

```diff
@@ -37,6 +37,7 @@ source_set("p3a") {
     "//brave/components/brave_prochlo:prochlo_proto",
     "//brave/components/brave_referrals/common",
     "//brave/vendor/brave_base",
+    "//chrome/common:common",
```

Comment:
do we still need `//brave/common` now?

Reply:
Yes, it's still needed because "//brave/components/p3a/brave_p3a_service.cc" still uses "//brave/common/brave_channel_info.h", and "brave_channel_info.cc" still needs to use "//chrome/common/channel_info.h".

```
ERROR at //brave/components/p3a/brave_p3a_service.cc:24:11: Can't include this header from here.
#include "brave/common/brave_channel_info.h"
          ^--------------------------------
The target:
  //brave/components/p3a:p3a
is including a file from the target:
  //chrome/common:common

```